src/components/Cardlist/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from "react";
+import Card from "../Card";
+import styles from "./index.module.css";
+
+interface IProps {
+  data: {
+    body_text: string;
+    header_text: string;
+    id: number;
+  }[];
+}
+
+const CardList: FC<IProps> = ({ data }) => {
+  console.log(data);
+  return (
+    <div className={styles.card_list}>
+      {data.map((e) => {
+        console.log(e.id);
```

Comment:
delete console.log, also wrap <Card/> in <div/> and pass key to <div/> to avoid each component should have key component warning